packages/nextjs/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@axiomhq/nextjs",
+  "version": "0.0.1",
+  "author": "Axiom, Inc.",
+  "license": "MIT",
+  "description": "The official Next.js package for Axiom",
+  "type": "module",
+  "main": "dist/cjs/index.cjs",
+  "contributors": [
+    "Lukas Malkmus <lukas@axiom.co>",
+    "Islam Shehata <islam@axiom.co>",
+    "Arne Bahlo <arne@axiom.co>",
+    "Gabriel De Andrade <gabriel@axiom.co>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/axiomhq/axiom-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/axiomhq/axiom-js/issues"
+  },
+  "homepage": "https://github.com/axiomhq/axiom-js/blob/main/packages/nextjs/README.md",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "vite build",
+    "format": "eslint . --fix",
+    "lint": "eslint .",
+    "test": "vitest run ./test/unit/*"
+  },
+  "keywords": [
+    "axiom",
+    "api",
+    "rest",
+    "client",
+    "axiom-nextjs",
+    "axiom sdk",
+    "axiom js",
+    "next-axiom",
+    "nextjs",
+    "logging"
+  ],
+  "dependencies": {
+    "@axiomhq/logging": "workspace:*"
+  },
+  "devDependencies": {
+    "@repo/eslint-config": "workspace:*",
+    "@tanstack/config": "^0.16.0",
+    "next": "15.1.4",
+    "vite": "^5.2.14",
+    "vitest": "^0.34.6"
+  },
+  "peerDependencies": {
+    "next": "15.1.4"
```

Comment:
Perhaps you can be more flexible with the versioning here, something like `>=15.1`